gravitee-apim-plugin/gravitee-apim-plugin-endpoint/gravitee-apim-plugin-endpoint-kafka/src/main/java/io/gravitee/plugin/endpoint/kafka/KafkaEndpointConnector.java

```diff
@@ -286,4 +286,12 @@ private String getKey(final MessageExecutionContext ctx, final Message message)
         }
         return key;
     }
+
+    @Override
+    public Connector stop() throws Exception {
+        super.stop();
+        KafkaSenderFactory.INSTANCE.clear();
```

Comment:
See above but clearing the shared map when stopping the connector will impacts all the other apis and connectors.

Reply:
already fixed.